desktop-exporter/app/utils/duration.ts

```diff
@@ -0,0 +1,37 @@
+import Timestamp from "timestamp-nano"
+
+export function getDuration(startTimestamp : string, endTimestamp:string){
```

Comment:
nit: odd spacing for the type signatures. Run prettier?

```suggestion
export function getDuration(startTimestamp: string, endTimestamp: string){
```